test/fixtures/get-orders-response.json

```diff
@@ -7,9 +7,9 @@
         "value": "17.70155237781915",
         "counterparty": "rMwjYedjc7qqtKYVLiAccJSmCwih4LnE2q"
       },
-      "totalPrice": {
+      "price": {
```

Comment:
I noticed the properties doesn't have `maker`.  So I guess this is basically a wrapper around `account_orders` ?